cpp/src/aztec/plonk/composer/composer_base.cpp

```diff
@@ -2,7 +2,7 @@
 #include <proof_system/proving_key/proving_key.hpp>
 #include <plonk/proof_system/utils/permutation.hpp>
 
-namespace waffle {
+namespace bonk {
```

Comment:
This is problematic because [here](https://github.com/AztecProtocol/barretenberg/blob/fdc97b6ff57d87161ed047a785ff98167c88deae/cpp/src/aztec/plonk/composer/composer_base.hpp#L14) we are declaring `proving_key` and `verification_key`, which are under the `bonk` namespace. If I change the namespace to `plonk`, conflicts arise which I could not resolve. For now, I am adding a TODO for this. Please let me know if you see a way out.

Reply:
We do not need a forward declaration of `proving_key` and `verification_key` in the `.hpp`. You can simply delete those two lines then change the namespace in the both cpp/hpp to plonk.

Reply:
Works. Thanks.